lib/quantum/normalizer.ex

```diff
@@ -110,6 +110,7 @@ defmodule Quantum.Normalizer do
       schedule: extract(:schedule, opts),
       task: extract(:task, opts),
       args: extract(:args, opts, []),
+      overlap: extract(:overlap, opts, Application.get_env(:quantum, :default_overlap, true)),
```

Comment:
I'm a bit confused on this fix. wont the defaults already be set within the `Quantum.Job` 's defstruct definition?  And if I'm missing something and they are not, should we have the other assorted opts like `args` defaulting to `Application.get_env` as well here?